.github/workflows/update_tag.yml

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# GH actions.
+# We use it to cover windows and mac builds
+# Jenkins is still the primary CI
+
+name: Update last-successful tag
+
+on:
+  workflow_dispatch:
+  schedule:
+    cron: 15/* * * * *
+
+concurrency:
+  group: update-last-successful-tag
+  cancel-in-progress: true
+
+jobs:
+  update-last-successful-tag:
+    runs-on: ubuntu-latest
```

Comment:
just wondering if we should be pinning these to avoid python version being bumped unexpectedly?

Reply:
we're not depending on any part of Python that would change but pinned to 20.04 just in case